README.md

```diff
@@ -1 +1,201 @@
+
+![app-look](https://user-images.githubusercontent.com/7433143/235880172-4fc320c0-c9bd-48d5-9673-21650b767bb6.jpg)
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- We can add, remove, update and display blogs.
+- It has user authentication.
+- It is well designed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [https://mysite-ji7l.onrender.com/]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```
+
+ - Laptop/Desktop.
+ - Internet.
+ - Ruby.
+ - Rails.
+ - Vs Code.
+
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+ - mkdir my-folder
+ - cd my-folder
+ - git clone git@github.com:SaboorSohaib/Blog-app.git
+ - cd Blog-app
+ - bundle install
+ - rails db:create
+ - rails db:migrate
+
+```
+### Usage
+
+To run the project, execute the following command:
+
+- rails server
+
+### Run tests
+
+- rspec.
+
+### Check Linter 
+
+- rubocop
+- npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+- Render
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Juan Diaz**
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Enhance design.** 
+- **Add API endpoits.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can fork it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+UI design by <a href="https://www.behance.net/gregoirevella">Gregoire Vella</a>.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The link to the MIT license is not working, please make sure it refers to the license as shown in the readme template provided by Microvese.